workshop/README.md

```diff
@@ -80,7 +80,6 @@ This file has the following attributes:
 | [Machine Learning with AutoAI](machine-learning-autoai/README.md) | Use AutoAi to quickly generate a Machine Learning pipeline and model |
 | [Deploy and Test Machine Learning Models](machine-learning-deployment-scoring/README.md) | Deploy and machine learning models using several approaches |
 | [Monitoring models with OpenScale GUI (Fastpath Monitoring)](openscale-fastpath/README.md) | Quickly deploy an OpenScale demo with FastPath |
-| [Monitoring models with OpenScale (Notebook)](openscale-notebook/README.md) | See the OpenScale APIs in a Jupyter notebook and manually configure the monitors |
```

Comment:
@scottdangelo We want to showcase the manual config (openscale-manual-config) this time around, no ?

Reply:
OK, I wasn't sure. Do you want me to delete FastPath, or do both?

Reply:
I would keep both.